sql/moz-fx-data-shared-prod/firefox_accounts_derived/docker_fxa_customs_sanitized_v2/query.sql

```diff
@@ -0,0 +1,20 @@
+-- Query for firefox_accounts_derived.docker_fxa_customs_sanitized_v2
+SELECT
+  DATE(@submission_date) AS `date`,
+  * REPLACE (
+    (
+      SELECT AS STRUCT
+        jsonPayload.* REPLACE (SHA256(jsonPayload.email) AS email, SHA256(jsonPayload.ip) AS ip)
+    ) AS jsonPayload
+  )
+FROM
+  `moz-fx-fxa-prod.gke_fxa_prod_log.stderr`
+WHERE
+  DATE(_PARTITIONTIME) BETWEEN
+    DATE_SUB(@submission_date, INTERVAL 1 DAY) AND @submission_date
```

Comment:
- I'd recommend also including the day after the submission date (if anything, that's more important than the day before, as slightly late arriving data is much more likely than clock skew in the opposite direction).
- I'd suggest putting parentheses around the `(... BETWEEN ... AND ...)` expression so `bqetl format` doesn't format it confusingly to look like the `AND ...` is a normal boolean condition.

Reply:
IMO this is a must-fix before approval.